rotkehlchen/api/rest.py

```diff
@@ -247,6 +247,34 @@ def api_response(
     return response
 
 
+def asynchronous() -> Callable:
```

Comment:
Probably is fine but for me `asynchronous` is a word that could mean many things in the context of the code. @LefterisJP might have another view but what do you think about using a name like `async_call`?

Reply:
Maybe, yeah. If I rename it to `async_api_call`, will that be fine from your perspective?

Reply:
yes, for me at least is clearer with that name what the wrapper is doing